core/src/banking_stage/consume_worker.rs

```diff
@@ -368,14 +383,30 @@ impl ConsumeWorkerMetrics {
     }
 }
 
-#[derive(Default)]
 struct ConsumeWorkerCountMetrics {
     transactions_attempted_execution_count: AtomicUsize,
     executed_transactions_count: AtomicUsize,
     executed_with_successful_result_count: AtomicUsize,
     retryable_transaction_count: AtomicUsize,
     retryable_expired_bank_count: AtomicUsize,
     cost_model_throttled_transactions_count: AtomicUsize,
+    min_prioritization_fees: AtomicU64,
+    max_prioritization_fees: AtomicU64,
+}
+
+impl Default for ConsumeWorkerCountMetrics {
+    fn default() -> Self {
+        Self {
+            min_prioritization_fees: AtomicU64::new(u64::MAX),
+            transactions_attempted_execution_count: AtomicUsize::default(),
+            executed_transactions_count: AtomicUsize::default(),
+            executed_with_successful_result_count: AtomicUsize::default(),
+            retryable_transaction_count: AtomicUsize::default(),
+            retryable_expired_bank_count: AtomicUsize::default(),
+            cost_model_throttled_transactions_count: AtomicUsize::default(),
+            max_prioritization_fees: AtomicU64::default(),
```

Comment:
minor: just keep the same order as declaration:
```suggestion
            transactions_attempted_execution_count: AtomicUsize::default(),
            executed_transactions_count: AtomicUsize::default(),
            executed_with_successful_result_count: AtomicUsize::default(),
            retryable_transaction_count: AtomicUsize::default(),
            retryable_expired_bank_count: AtomicUsize::default(),
            cost_model_throttled_transactions_count: AtomicUsize::default(),
            min_prioritization_fees: AtomicU64::new(u64::MAX),
            max_prioritization_fees: AtomicU64::default(),
```